twisted/spread/jelly.py

```diff
@@ -68,9 +68,15 @@
 import decimal
 from functools import reduce
 import copy
-
 import datetime
 
+try:
+    from types import (ClassType as _OldStyleClass,
+                       InstanceType as _OldStyleInstance)
+except ImportError:
+    _OldStyleClass = None
+    _OldStyleInstance = None
```

Comment:
Nice!  I updated the diff to use an empty tuple, and now a whole bunch of
jelly tests are passing on Python 3